### PR TITLE
Интеграция нейронки в джанго

### PR DESCRIPTION
**Pull request by OsipovaValeria**
Осипова Валерия

--> Сделан эндпоинт, который принимает картинку, подставляет в модель, и отдает параметры модели, предсказания (метка класса) + вероятности для каждой модели, и предсказание ансамбля.

--> Пример выходных данных:

{'params': {'optimizer': 'Adamax', 'learning_rate': 0.001, 'loss': 'categorical_crossentropy', 'metrics': 'accuracy'}, 
'individual_predictions': [('MEL', array([0.0017688 , 0.00333721, 0.00279451, 0.0018594 , 0.981265  ,
       0.00374212, 0.00210421, 0.00312868], dtype=float32)), ('MEL', array([0.00167712, 0.00200976, 0.00271847, 0.00154107, 0.9843498 ,
       0.00249063, 0.00222453, 0.00298872], dtype=float32)), ('MEL', array([0.00286707, 0.00875229, 0.00569259, 0.00337079, 0.9439307 ,
       0.01030423, 0.00536121, 0.0197212 ], dtype=float32))], 
'ensemble_prediction': ('MEL', array([0.006313  , 0.01409926, 0.01120557, 0.00677127, 2.9095454 ,
       0.01653697, 0.00968995, 0.02583859], dtype=float32))}

### CODE REVIEW

**Comment by OsipovaValeria** [comment]
Добавила инициализацию нейронки в backend

**Comment by reznikovkg** [comment]
Можете продемонстрировать работу в видео? Как вы тестировали работу эндпоинта?

**Comment by OsipovaValeria** [comment]
У меня есть демонстрационный скрипт, который повторяет последовательность логических шагов, необходимых для обработки на серверной стороне - подготовка и загрузка нейронок, передача изображения на обработку и получение результата от модели

https://github.com/user-attachments/assets/b1f4122d-afa9-46a6-adf5-95a34de0a086

**Comment by reznikovkg** [comment]
По какому маршруту работает эндпоинт? Продемонстрировать работу, можно например через postman или curl

**Comment by OsipovaValeria** [comment]
https://github.com/user-attachments/assets/be48e50b-03f1-48d8-8c0f-d55c2775bb63
